src/sisl/viz/data_sources/atom_data.py

```diff
@@ -66,15 +66,117 @@ def AtomNOrbitals(geometry, atoms=None):
 
 
 class AtomDefaultColors(AtomData):
+    # Taken from Jmol (https://jmol.sourceforge.net/jscolors/)
     _atoms_colors = {
```

Comment:
would it make sense to make this a `defaultdict`?

Reply:
Why?

Reply:
It's used like this, right below the dictionary:

```python
def function(self, geometry, atoms=None):
        return np.array(
            [
                self._atoms_colors.get(atom.symbol, self._atoms_colors["else"])
                for atom in geometry.sub(atoms).atoms
            ]
        )
```

Reply:
Hmm but a `defaultdict` is meant to initialize objects when the key is not there. Isn't it a bit overkill for this?

Reply:
Not a problem then, it would make its use simpler since one shouldn't care if the key is there or not. I'll merge.

Reply:
I can convert it to something like `self._atoms_colors.get( atom.symbol, self._fallback_color)` if it looks better.

Reply:
> Not a problem then, it would make its use simpler since one shouldn't care if the key is there or not. I'll merge.

Ok, anyway this can be modified whenever we want :)